admin_tools/dashboard/templates/admin/app_index.html

```diff
@@ -1,19 +1,9 @@
-{% extends "admin/index.html" %} 
-{% load i18n admin_tools_dashboard_tags %} 
-
-{% if not is_popup %}
-
+{% extends "admin:admin/app_index.html" %} 
+{% load admin_tools_dashboard_tags %} 
+{% block extrastyle %}
```

Comment:
I can confirm this but I recall that the extrastyle block has issues. I think Form.media is rendered after it - meaning that you can't use css load order to override any css that lives in Django's form Media css - thus forcing you to use lots of nasty !important declarations.

It's a bit counterintuitive but I've used {% block bots %} for css as this is guaranteed to come last.

I'll check the accuracy of this when I have a moment but if that's correct do you have a problem using the bots block?